src/application.rs

```diff
@@ -16,6 +16,8 @@ use tonic::{
 };
 use tracing::debug;
 
+const CHUNK_SIZE: usize = 1 * 1024 * 1024; // use 1MB chunks
```

Comment:
Can we make this an extension? See https://github.com/CoLearn-Dev/colink-sdk-rust-dev/tree/main/src/extensions for examples

Reply:
Done.